db/schema.rb

```diff
@@ -0,0 +1,37 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20170502015702) do
+
+  create_table "customers", force: :cascade do |t|
```

Comment:
hmmmm that's a great question that i do not know the answer to! i imagine the answer is yes, they do need to be named properly, but i could 100% be wrong about that. this is definitely worth looking up, but i don't think it should be a priority.

is your goal to just name them differently? or are you importing data based on these columns?